packages/core/src/components/scroll/scroll.tsx

```diff
@@ -106,15 +103,11 @@ export class Scroll {
         // compute relative movement between these two points
         var movedTop = (positions[startPos - 2] - positions[endPos - 2]);
         var movedLeft = (positions[startPos - 1] - positions[endPos - 1]);
-        var factor = 16.67 / (positions[endPos] - positions[startPos]);
+        var factor = 16.67 / (positions[startPos] - positions[endPos]);
```

Comment:
Notice movesTop and movedLeft are calculared correctly (startPos-endPos), but the elapsed timestamp is calculated  (endPos-startPos). Resulting in negated values of the velocity,